testing-guidance/workflow-tests/o3de-project-manager/remote-projects/remote-projects-workflow-tests.md

```diff
@@ -4,7 +4,12 @@ These workflows center around testing the basic functionality of the Project Man
 
 ## Prerequisites
 
-*   Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
+* Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
+* A GitHub repository that has a compatible remote project (older projects may need to be updated to support this feature).
```

Comment:
I suspect this doesn't have anything to do with older projects, rather that any project you want to host needs a repo.json.

Project Manager reads the repro.json to find out where to look for the remote project's project.json.
The project.json contains information about the project's git url; this is what the Project Manager will use to git clone.
```
    "origin": "https://github.com/o3de/o3de-NewspaperDeliveryGame",
    "origin_uri": "https://github.com/o3de/NewspaperDeliveryGame.git",
```
 
https://github.com/o3de/NewspaperDeliveryGame/blob/main/repo.json
https://github.com/o3de/NewspaperDeliveryGame/blob/main/project.json

I havent dug into the code myself, just guessing.